### PR TITLE
Prettier fy code

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: frontend/.prettierignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
-node_modules
```
Comment (tiagojsag) [inline frontend/.prettierignore]: https://prettier.io/docs/en/cli.html#with-node-modules

File: frontend/scripts/actions/app.actions.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ export function resize() {
 }
 
 export function toggleMap() {
-  return (dispatch) => {
```
Comment (tiagojsag) [inline frontend/scripts/actions/app.actions.js]: from here on, it's all prettier changes

File: frontend/.eslintrc.json [hunk 3]
```diff
@@ -26,30 +25,14 @@
         "components": ["a"]
       }
     ],
-    "indent": [
```
Comment (tiagojsag) [inline frontend/.eslintrc.json]: as per the new "eslint-check" (which is part of the online documentation of one of those eslint-prettier-config-foo modules) these rules conflict with the prettier config

**Comment by sorodrigo** [comment]
Tested and looks good to go

**Pull request merged**